tango_simlib/tests/test_sim_test_interface.py

```diff
@@ -420,7 +421,7 @@ def test_StopQuantitySimulation_command(self):
         for quantity_name in expected_result.keys():
             self.assertIn(quantity_name, device_attributes)
 
-        self.sim_control_device.command_inout(command_name, expected_result.keys())
+        self.sim_control_device.command_inout(command_name, list(expected_result))
```

Comment:
why did it change from `expected_result.keys()` to just `expected_result`

Reply:
it is a compatibility change.
see the following link. https://python-future.org/compatible_idioms.html#dict-keys-values-items-as-a-list

Reply:
Cool thank you.